code/game/world.dm

```diff
@@ -212,6 +207,14 @@ var/world_topic_spam_protect_time = world.timeofday
 
 	..(reason)
 
+/world/proc/send_reboot_sound()
+	var/reboot_sound = SAFEPICK(reboot_sfx)
+	if(reboot_sound)
+		var/sound/reboot_sound_ref = sound(reboot_sound)
+		for(var/client/C as anything in GLOB.clients)
```

Comment:
change C to be something more descriptive. 'client' as a variable name is fine too